cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -84,6 +84,8 @@ spec:
             - --container=heapster
             - --poll-period=300000
             - --estimator=exponential
+          securityContext:
+            runAsNonRoot: false
```

Comment:
> Actually, I don't understand this statement. runAsNonRoot: false means they explicitly reserve the right to run as root if the image dictates it, right?

Yes, this field corresponds with a `MustRunAsNonRoot` strategy and acts as a marker for the kubelet to know it should check the image.  This is for the case of a nil `runAsUser`, the `MustRunAsNonRoot` strategy itself handles any non-nil `runAsUser` cases during admission.  Shouldn't need to set it to false, it's a bug if you must.